src/ws-router.js

```diff
@@ -0,0 +1,37 @@
+var setPrototypeOf = require('setprototypeof');
+
+module.exports = WsRouter;
+
+function WsRouter (options) {
+	if (!(this instanceof WsRouter)) {
+		return new WsRouter(options);
+	}
+
+	function wsRouter (req, socket, head, next) {
+		wsRouter.handle(req, socket, head, next);
```

Comment:
These arguments don't seem to match the ones below on line 24?  What is going on here?